src/main/webapp/WEB-INF/tags/answerWith.tag

```diff
@@ -7,7 +7,7 @@
 <%@attribute name="commentVotes" type="org.mamute.model.CommentsAndVotes" required="true" %>
 <section class="post-area ${answer.isVisibleForModeratorAndNotAuthor(currentUser.current) ? 'highlight-post' : '' }">
 	<div class="post-meta">
-		<tags:voteFor item="${answer}" type="${t['answer.type_name']}" vote="${vote}"/>
+		<tags:voteFor item="${answer}" type="${answer.typeName}" vote="${vote}"/>
```

Comment:
@cpfeiffer: I just noticed that this breaks up/down-voting (at least for questions)!
data-type inside voteFor.tag has to be the localized name of the type.

This patch (on top of yours) fixes voting again: https://paste.kde.org/pxi41fsea